components/script/dom/window.rs

```diff
@@ -702,41 +702,43 @@ impl WindowMethods for Window {
     }
 
     // https://html.spec.whatwg.org/multipage/#dom-opener
-    fn Opener(&self, cx: JSContext, in_realm_proof: InRealm) -> JSVal {
```

Comment:
Huh, the missing InRealm argument looks like a codegen bug to me. I'll file a followup issue about it.

Reply:
indeed it is weird, I just added `Throws` and `in_realm_proof ` was removed from codegen!